tutorials/5-docker-intro/index.md

```diff
@@ -0,0 +1,288 @@
+---
+parent:
+  title: Docker Introduction
+  description: Get started with Docker
+  number: 
+tags:
+  - tutorial
+  - dev-ops
+order: 0
+title: Docker introduction
+---
+
+<div class="tm-overline tm-rf-1 tm-lh-title tm-medium tm-muted">Know your way around Docker</div>
+<h1 class="mt-4 mb-6">Docker Introduction</h1>
+
+[Docker](https://www.docker.com) is a kind of virtualization, with performance benefits.
+
+## Concepts
+
+Being different, Docker introduces new concepts with their own vocabulary.
```

Comment:
Different from what?

Reply:
In the field of computer virtualisation, being different from classical virtualisation...

something like that...